spec/index.bs

```diff
@@ -620,8 +620,9 @@ interface GPUComputePassEncoder : GPUProgrammablePassEncoder {
 
 
 enum GPULoadOp {
-    "clear",
-    "load"
+    "load",
+    "opaqueBlack",
+    "zero"
```

Comment:
opaqueBlack and zero are up for discussion.

Reply:
Prefer not to duplicate functionality of GPUColor.